cfme/containers/container.py

```diff
@@ -41,6 +41,17 @@ def get_detail(self, *ident):
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
 
+    @staticmethod
+    def get_container_names():
+        navigate_to(Container, 'All')
+        # sel.force_navigate('containers_containers')
+        return map(lambda r: r.name.text, list_tbl.rows())
```

Comment:
please use list comprehensions here for future-safety and readability

`[r.name.text for r in list_tbl.rows()]` reads better and is sure to materialize on python3

similar changes can be done below